pkg/scraper/scraper.go

```diff
@@ -81,7 +83,7 @@ func RegisterScraperMetrics(registrationFunc func(metrics.Registerable) error) e
 	return nil
 }
 
-func NewScraper(nodeLister v1listers.NodeLister, client KubeletInterface, scrapeTimeout time.Duration) *scraper {
+func NewScraper(nodeLister v1listers.NodeLister, client client.KubeletMetricsInterface, scrapeTimeout time.Duration) *scraper {
```

Comment:
May be missing client.KubeletMetricsInterface?

Reply:
Yep, forgot to `git add` file :P